core/UrlHelper.php

```diff
@@ -33,9 +33,9 @@ class UrlHelper
     * @param $test String to test.
     * @param $patterns Array of strings or regexs.
     *
-    * @return true if $test matches or is equal to one of the regex/string in $patterns, false otherwise.
+    * @return bool true if $test matches or is equal to one of the regex/string in $patterns, false otherwise.
     */
-    protected static function in_array_matches_regex($test, $patterns)
+    protected static function inArrayMatchesRegex($test, $patterns): bool
```

Comment:
Technically a BC since custom code can extend the class and override the method?

Reply:
Theoretically yes. The class or method is not marked as `@api` and it's at least not used in our plugins, so guess we could do that nevertheless.